### DISCUSSION TITLE
Getting PV5 and PV6 for H3-Pro

### DISCUSSION THREAD

**Discussion by dagroe** [nathanmarlor/foxess_modbus#695] [Q&A] [answered]
Hi all,

I have my H3-Pro running with the integration and modbus via TCP. Thanks for this awesome integration!

I have one issue: My strings are connected on PV1, PV3, and PV5, but I can only see PV1, PV2, PV3, and PV4. Is there something I can configure to access the two missing ones?

Thanks

**Comment by dagroe** [comment]
I can read the values (e.g. power) via the devtools in the registers following the PV4 registers, e.g. PV5 power at 39287

**Comment by dagroe** [comment, marked as answer]
I added a PR here, which adds the entities: #696

**Reply by canton7** [reply]
Thanks for getting that in!
